scout/server/blueprints/variants/forms.py

```diff
@@ -105,14 +105,17 @@ class VariantFiltersForm(FlaskForm):
     load_filter = SubmitField(label="Load filter")
     delete_filter = SubmitField(label="Delete filter")
 
-    chrom = SelectField("Chromosome", [validators.Optional()], choices=CHROMOSOME_OPTIONS)
+    chrom = SelectField(
+        "Chromosome", [validators.Optional()], choices=CHROMOSOME_OPTIONS, default=""
```

Comment:
Was that it?!

Reply:
It's the default option with All chromosomes, look at line 33 :
`CHROMOSOME_OPTIONS = [("", "All")] + [(chrom, chrom) for chrom in CHROMOSOMES]`

Reply:
> It's the default option with All chromosomes, look at line 33 :
> `CHROMOSOME_OPTIONS = [("", "All")] + [(chrom, chrom) for chrom in CHROMOSOMES]`

Yeah, agree, thats the way it should be - just dont quite see the connection to the clinical filter, which basically has its own set of parameters. But ah - perhaps not for chromosome?

Reply:
Nope, chrom was None

Reply:
One could also set that explicitly in the clinical filter dict. But this deals with multiple potential issues, so lets leave it at that! Well found!